recipe/meta.yaml

```diff
@@ -23,30 +22,34 @@ requirements:
     - pip
     - python
     - flit-core >=3.2,<4
-    - setuptools
-    - wheel
   run:
     - python
-    - numpy >=1.17,!=1.24.0
-    # matplotlib 3.6.1 incompatible with latest master for 0.12.0
-    # https://github.com/mwaskom/seaborn/issues/3072
-    - matplotlib-base >=3.1,!=3.6.1
-    - pandas >=0.25
-    - typing_extensions  # [py<38]
+    - numpy >=1.20,!=1.24.0
+    - matplotlib-base >=3.4,!=3.6.1
+    - pandas >=1.2
   run_constrained:
-    - scipy >=1.3
-    - statsmodels >=0.10
+    - scipy >=1.7
+    - statsmodels >=0.12
 
 test:
+  source_files:
+    - tests
+    - pyproject.toml
```

Comment:
`pyproject.toml` isn't mentioned in any test file https://github.com/search?q=repo%3Amwaskom%2Fseaborn+%22pyproject.toml%22&type=issues. I think it can be removed

Reply:
I included it for [these lines](https://github.com/mwaskom/seaborn/blob/f0b48e891a1bb573b7a46cfc9936dcd35d7d4f24/pyproject.toml#L66-L71).

Reply:
Removing warnings won't help building the package. It's up to you remove the file or proceed with it